language-support/java/bindings-rxjava/src/main/java/com/daml/ledger/rxjava/TimeClient.java

```diff
@@ -12,6 +12,8 @@
 public interface TimeClient {
 
     Single<Empty> setTime(Instant currentTime, Instant newTime);
+    Single<Empty> setTime(Instant currentTime, Instant newTime, String accessToken);
```

Comment:
Are we duplicating the interface methods for backwards compatibility? We may want to simply add a method with `Optional` access token and deprecate the old one.

Reply:
I'd prefer keeping backwards compatibility unless strictly needed, but thanks for the feedback.